src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -133,6 +124,29 @@ protected override void ProcessRecord()
             }
         }
 
+        /// <summary>
+        /// Calculate the hash data from the Stream with the specified hash algorithm
+        /// </summary>
+        /// <returns>The computed hash data</returns>
+        private byte[] ComputeHash(Stream stream)
+        {
+            switch (Algorithm)
```

Comment:
Use C# 8.0 switch expression.

Reply:
I have kept the old switch expression for now, because in the C# 8 switch expression I can't use Assert.Debug() _and_ return SHA256 hash as default.